datadog_checks_dev/datadog_checks/dev/structures.py

```diff
@@ -0,0 +1,37 @@
+# (C) Datadog, Inc. 2018
+# All rights reserved
+# Licensed under a 3-clause BSD style license (see LICENSE)
+import abc
+import os
+
```

Comment:
?

Reply:
Used to separate std vs 3rd party libs I believe. I usually do this as well.

Reply:
hum didn't realized it was a 3rd party lib